imsv-docs-astro/src/content/docs/guides/authentication.mdoc

```diff
@@ -28,8 +28,11 @@ Target account:
   omitted if the target is the root account (such as when creating an account
   for example)
 
-The caller must have the correct permissions over this account to perform the
-requested operation.
+The caller must have a role with the correct permissions over this account to perform the
+requested operation. We have two roles for API Key authentication:
+
+- **account-admin** - Used for manual resource provisioning steps, eg: creating funding channels, client applications
```

Comment:
I'm not so sure about the word "manual", it feels out of context when we're describing API calls. Maybe it's better to describe these as one-time provisioned resources or something like that? with the accent that it won't be done repeatedly like card issuing

Reply:
Totally agree @catluin , thanks for the feedback 🙌 

The one-time thing may be true at the moment, but I imagine we'll eventually allow people to edit account preferences using the account admin key.

I changed it to this, how does this sound?

"Used for provisioning account resources, eg: creating funding channels, client applications"